framework-cloud/framework-openshift-strimzi/src/main/java/org/kie/cloud/strimzi/resources/KafkaClusterBuilder.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.cloud.strimzi.resources;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class KafkaClusterBuilder {
+    private String name;
+    private int kafkaReplicas = 3;
+    private Map<String, String> kafkaConfig = new HashMap<>();
+    private int zookeeperReplicas = 3;
+    private boolean persistent = false;
+
+    public KafkaClusterBuilder(final String name) {
+        this.name = name;
+    }
+
+    public KafkaClusterBuilder withKafkaReplicas(final int kafkaReplicas) {
+        this.kafkaReplicas = kafkaReplicas;
+
+        return this;
+    }
+
+    public KafkaClusterBuilder addKafkaConfigItem(final String key, final String value) {
```

Comment:
can be also handy to have method 'addKafkaConfigItems(Map<String, String>)' to add all items from map. Later in code are set same properties 3 times. To avoid configuration issue it would be better to add these same values to one map and then set it to all 3 topics.